{{cookiecutter.project_name}}/pyproject.toml

```diff
@@ -60,6 +61,19 @@ packages = [ "src/{{ cookiecutter.package_name }}" ]
 
 {% endif -%}
 
+[tool.hatch.envs.default]
+installer = "uv"
```

Comment:
Naive question: What happens if `uv` is not available?

Reply:
[Mu](https://csarven.ca/mu), Hatch comes with an internal version of uv, so it’s always available.